target/consensource-ui
# feat: created claimable header component

## Proposed change/fix

- Creates a `<ClaimableDataPageHeader />` component that can be reused across pages.
- Refactors cert and factory profile pages to use this component
- Adds claimed/unclaimed info to cert card
- Creates a utility function to safely check if an object has a property
- Creates an `icons` file to encourage consistent use of icons 

<img width="1426" alt="Screen Shot 2020-10-20 at 1 59 35 PM" src="https://user-images.githubusercontent.com/20157849/96631898-8c3f5d00-12dc-11eb-872a-94478208d31f.png">
<img width="1439" alt="Screen Shot 2020-10-20 at 1 59 42 PM" src="https://user-images.githubusercontent.com/20157849/96631905-8e092080-12dc-11eb-85f8-b91ab4258bff.png">


## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- Follow [these instructions](https://github.com/target/consensource-ui/pull/56) to create a valid cert
- Visit both the factory profile
- Verify that the claimed icon button is in the header
- Verify that the claimed icon button is in the cert card
- Visit the cert page
- Verify that the claimed icon button is present

``` yarn test src/view/components/Claims/__tests__/ClaimableDataPageHeader.spec.tsx```
